lib/cdo/hamburger.rb

```diff
@@ -0,0 +1,77 @@
+class Hamburger
+  def self.get_visibility(options)
+    show_teacher_options = "hide-always"
```

Comment:
Maybe `"hide-always"`, `"show-always"` and `"show-mobile"` should be constants, or an enum?